src/jest-extensions.js

```diff
@@ -1,4 +1,12 @@
-import {matcherHint, printReceived, printExpected} from 'jest-matcher-utils' //eslint-disable-line import/no-extraneous-dependencies
+//eslint-disable-next-line import/no-extraneous-dependencies
+import {
+  matcherHint,
+  printReceived,
+  printExpected,
+  stringify,
+  RECEIVED_COLOR as receivedColor,
```

Comment:
Just wondering here, can't we use `printReceived`? Is that `RECEIVED_COLOR` does something apart from `printReceived`?

Reply:
Yes. `printReceived` uses `RECEIVED_COLOR` internally, but it also stringifies the argument. But in the new code I added I wanted to show `attribute-name="value"` all in the received or expected color, without stringifying all of it. If I used `printReceived`/`printExpected` I'd have the whole `attribute-name="value"` itself surrounded by double quotes (and possibly those inner quotes escaped then).